conferences/jsconf.yaml

```diff
@@ -103,3 +103,10 @@
     name: JSConf China
     site: http://2017.jsconf.cn
     logo: images/jsconf_cn.png
+    
+# JSConf Korea
```

Comment:
Please put yourself in position 4 with the other events that have confirmed future dates.

Reply:
I changed it!